### DISCUSSION TITLE
I can't verification the domain.

### DISCUSSION THREAD

**Discussion by fscarmen** [vercel/community#3684] [Help] [answered]
### Page to Investigate

cdn-gh.vercel.app

### Steps to Reproduce

I followed the official instructions but still cannot verify the domain, here is a screenshot of the operation.

<img width="1478" alt="image" src="https://github.com/vercel/community/assets/62703343/08903ecf-ca24-434d-8e9e-4d650774d923">

![image](https://github.com/vercel/community/assets/62703343/b77e9c6b-b4e8-4561-82f3-f35583abe4d1)

**Comment by amyegan** [comment, marked as answer]
Hey there. This is because `cloudflare.now.cc` is a subdomain. The TXT record would need to be added to the `now.cc` domain as directed in the domain verification instructions.

**Comment by fscarmen** [comment]
Ths.
